scripts/README.rst

```diff
@@ -0,0 +1,84 @@
+This module provides scripts to update the STM32Cube for Zephyr.
+Follow below steps to update the STM32Cube HAL from the latest
+version found in STM32CubeXX repositories.
+################################################################
+
+Step 1: Modify the code
+***********************
+
+In order to update all the STM32CubeXX versions at once, run::
+
+    $ python3 ./update_stm32_package.py
+
+Alternatively, in order to update STM32CubeXX versions one by one, run for example::
+
+    $ python3 ./update_stm32_package.py -s stm32f1
+
+check STM32CubeXX changes
+=========================
+Check that all existing change to modules are correctly reported
+in the new hal_stm32 version
+
+Update each modified stm32cube/stm32XXxx/README in the last section,
+
+Remove the following line "--> please check that the following list is still valid:"
+
+	Patch List:
+	--> please check that the following list is still valid:
+
+
+Step 2: Open a pull requests
+****************************
+
+hal_stm32 PR
+============
+Open a single pull request with 1 commit per stm32 familly (stm32xx)
+in repo https://github.com/zephyrproject-rtos/hal_stm32
+
+
+Modify the west.yml
+===================
+In order to benefit from this PR in zephyr, modify zephyrproject/zephyr/west.yml
```

Comment:
```suggestion

In order to benefit from this PR in zephyr, modify zephyrproject/zephyr/west.yml
```

Reply:
Would probably be better to refer official docs here instead of giving a detailed explanation: https://docs.zephyrproject.org/latest/guides/modules.html#process

Reply:
I prefer to keep like this to show the diiference betwwen pointing to PR
```
	    - name: hal_stm32
	      revision: pull/60/head
	      path: modules/hal/stm32
```
and pointing to commit
```
	    - name: hal_stm32
	      revision: a813cd83b0cfbaaa625f4941d04baa3f93c37476
	      path: modules/hal/stm32
```